### PR TITLE
cleanUp and made mouse a pointer for infosign

### PR DESCRIPTION
**Pull request by paolovic**

### CODE REVIEW

File: src/components/views/game/Game.js [hunk 1]
```diff
@@ -111,7 +111,7 @@ const Game = ({id}) => {
 
             mapData.forEach((row, y) => {
                 mapArray.push([]);
-                row.forEach((tile, x) => {
+                row.forEach((x) => {
```
Comment (apple00juice) [inline src/components/views/game/Game.js:114]: The first parameter in forEach is the value, but we need the index here which is the second parameter.

**Review by apple00juice — APPROVED** [review]
The rest looks good! 👍 tyvm

File: src/components/views/game/Game.js (hunk 1 above)
Comment (paolovic) [reply src/components/views/game/Game.js]: thx

File: .vscode/settings.json [hunk 2]
```diff
@@ -1,4 +1,4 @@
 {
     "workbench.preferredHighContrastColorTheme": "Tomorrow Night Blue",
-    "workbench.colorTheme": "Default High Contrast"
+    "workbench.colorTheme": "Default Dark+"
 }
```
Comment (luis-tm) [inline .vscode/settings.json:4]: what's this file?

**Review by luis-tm — APPROVED** [review]

**Review by mkorob — APPROVED** [review]

File: .vscode/settings.json (hunk 2 above)
Comment (paolovic) [reply .vscode/settings.json:4]: rubbish

**Pull request merged**